catalog/imaging/imaging-app/src/main/resources/features.xml

```diff
@@ -16,6 +16,12 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <feature name="imaging-render-plugin" install="auto" version="${project.version}"
```

Comment:
This is really unrelated to this PR but why are these features named with imaging instead of NITF which is what is in all of their descriptions and bundle names?